### PR TITLE
Completed login page

### PR DESCRIPTION
**Pull request by gina-wang-1021**

### CODE REVIEW

File: lib/views/logInView.dart [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                                style: TextStyle(
+                                  fontSize: 24.0,
+                                  fontFamily: 'Serif',
+                                  color: 'F5F5DC'.toColor(),
+                                )
+                            )
+                        )
+                      ])
+              ),
+            ]
```
Comment (ining310) [inline lib/views/logInView.dart:164]: image
overflow error

**Review by ining310 — COMMENTED** [review]
Have you tried to type Chinese account in the textField?  I can see abc option, but no Chinese.
Look nice.

File: lib/views/logInView.dart (hunk 1 above)
Comment (gina-wang-1021) [reply lib/views/logInView.dart:164]: fixed! thank u:)

**Comment by gina-wang-1021** [comment]
> Have you tried to type Chinese account in the textField? I can see abc option, but no Chinese. Look nice.

haven't tried using Chinese, but seems like we're only using alphabets and numbers in this case!

**Comment by ining310** [comment]
> > Have you tried to type Chinese account in the textField? I can see abc option, but no Chinese. Look nice.
> 
> haven't tried using Chinese, but seems like we're only using alphabets and numbers in this case!

no problem!

**Review by ining310 — APPROVED** [review]
LGTM👍

**Pull request merged**